packages/default-theme/src/components/organisms/SwFooterService.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <SwFooterNavigationColumn     
+    v-if="navigationElements && navigationElements.length"
+    :key="navigationElements[0].id" 
+    :category="navigationElements[0]" 
```

Comment:
please move key and category to the `setup` method as computed properties, using optional chaining there and will be ready 💪🏻